js/components/dashboard/activity-answers.js

```diff
@@ -1,10 +1,13 @@
 import React, { PureComponent } from "react";
 import ProgressBar from "./progress-bar";
-import Answer from "./answer";
+import Answer from "../../containers/dashboard/answer";
 
 import css from "../../../css/dashboard/activity-answers.less";
 
 export default class ActivityAnswers extends PureComponent {
+
+  studentAnswer;
```

Comment:
Is this an unused property? 

Possibly a typo.

Reply:
Thanks, fixed now.